### DISCUSSION TITLE
District heating network analysis

### DISCUSSION THREAD

**Discussion by Mamun5001** [oemof/tespy#439] [General]
I am a new user of tespy, I am looking for help everyone.
I want to analyze District heating network using tespy. 
What should be my steps?
I am requesting you to share some ideas.

**Comment by fwitte** [comment]
Hi,

could you share a little bit more context or do you have concrete questions? It is difficult to help with just about no information... there is a tutorial for a small scale heating system in the docs, maybe start with that.

Best

**Reply by Mamun5001** [reply]
Thank you for your reply. 
Now I want to investigate what happens, in case the 

- ambient temperature changes.
- heat load varies.
- The overall temperature level in the heating system is reduced.

I am trying to find out heat loss and efficiency in these 3 different conditions for my network.  I have taken the network which is solved already, sharing the link 
(https://github.com/oemof/oemof-examples/blob/master/oemof_examples/tespy/district_heating/dhs.py)

Can you write the code for this condition only, it will help me a lot to learn it. Thank you again.
